theme/js/components/browser.js

```diff
@@ -25,8 +25,8 @@ export default class Browser {
       tabs.removeClass(ac).find('a').attr('aria-selected', 'false');
       set(`browser.selectedTabIndex`, tabs.index(tab));
       tab.addClass(ac).find('a').attr('aria-selected', 'true');
-      this._tabPanels.removeClass(ac);
-      this._tabPanels.filter(link.attr('href')).addClass(ac);
+      this._tabPanels.removeClass(ac).removeAttr('aria-current');
+      this._tabPanels.filter(link.attr('href')).addClass(ac).attr('aria-current', true);
```

Comment:
Is het toepassen van een `aria-current` genoeg, voor de meeste gangbare screenreaders? Of is het ook nog wenselijk een `sr-only` tekstje "geselecteerd" op te nemen?

Reply:
Goeie vraag, ik heb overgenomen wat er in het oorspronkelijke issue vermeld staat. Ik nam eigenlijk aan dat hier al eerder over nagedacht was.

Reply:
Is `aria-current` en `.sr-only` niet dubbelop? Leest een screenreader met aria-current ondersteuning dan dubbel de geselecteerde state op?

Reply:
Zie de comment van @timveld. Dit is goed zo: alleen aria-current.